heft-plugins/heft-sass-plugin/src/SassTypingsGenerator.ts

```diff
@@ -3,7 +3,7 @@
 
 import * as path from 'path';
 import { render, Result, SassError } from 'node-sass';
-import postcss from 'postcss';
+import * as postcss from 'postcss';
```

Comment:
@iclanton I'm not sure this is an improvement, since it is renaming the APIs to be different from the official names of `postcss`.

`postcss.default()` is certainly a clumsy name for an API, but if library maintainers are still using `default` exports in the year 2022, then that is the API name that they chose. 🤷‍♂️